### PR TITLE
w: add Spanish translation

### PR DESCRIPTION
**Pull request by GustavoHiguera**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.es/linux/w.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# w
+
+> Muestra quien ha iniciado sesión y sus procesos.
+> Más información en: <https://www.geeksforgeeks.org/w-command-in-linux-with-examples/>.
```
Comment (kbdharun) [inline pages.es/linux/w.md:4]:
```suggestion
> Más información: <https://www.geeksforgeeks.org/w-command-in-linux-with-examples/>.
```

File: pages.es/linux/w.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+- Muestra información sobre todos los usuarios que han iniciado sesión actualmente:
+
+`w`
+
+- Muestra información sobre un usuario específico:
+
+`w {{user}}`
```
Comment (navarroaxel) [inline pages.es/linux/w.md:12]:
```suggestion
`w {{nombre_de_usuario}}`
```

**Review by navarroaxel — COMMENTED** [review]
This is an old page and the English version needed an update

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**